src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -123,15 +125,17 @@ private AnomalyDetectorSettings() {}
             Setting.Property.Dynamic
         );
 
-    public static final Setting<TimeValue> COOLDOWN_MINUTES = Setting
+    public static final Setting<TimeValue> AD_COOLDOWN_MINUTES = Setting
         .positiveTimeSetting(
             "plugins.anomaly_detection.cooldown_minutes",
             LegacyOpenDistroAnomalyDetectorSettings.COOLDOWN_MINUTES,
             Setting.Property.NodeScope,
             Setting.Property.Dynamic
         );
 
-    public static final Setting<TimeValue> BACKOFF_MINUTES = Setting
+    // Use TimeSeriesSettings.BACKOFF_MINUTES
+    @Deprecated
+    public static final Setting<TimeValue> AD_BACKOFF_MINUTES = Setting
```

Comment:
deprecated for next PR or is the idea to keep this because this is in use? Either answer works, just curious.

Reply:
It's latter.